src/ui/map.js

```diff
@@ -203,7 +203,7 @@ const defaultOptions = {
  * @param {number} [options.pitch=0] The initial pitch (tilt) of the map, measured in degrees away from the plane of the screen (0-60). If `pitch` is not specified in the constructor options, Mapbox GL JS will look for it in the map's style object. If it is not specified in the style, either, it will default to `0`.
  * @param {LngLatBoundsLike} [options.bounds] The initial bounds of the map. If `bounds` is specified, it overrides `center` and `zoom` constructor options.
  * @param {Object} [options.fitBoundsOptions] A [`fitBounds`](#map#fitbounds) options object to use _only_ when fitting the initial `bounds` provided above.
- * @param {boolean} [options.renderWorldCopies=true]  If `true`, multiple copies of the world will be rendered, when zoomed out.
+ * @param {boolean} [options.renderWorldCopies=true]  If `true`, multiple copies of the world will be rendered side by side when the map is zoomed out far enough that a single representation of the world does not fill the map's entire container.
```

Comment:
I know this was an issue with the docs even before you're change here, but technically it's not just zoomed out. zoomed right in to z14 looking showing a map of Fiji you want renderWorldCopies to true otherwise half the country will be cut off.

Perhaps a better way instead of saying when zoomed out is, saying outside of the -180 to 180degrees longitude bounds?

Reply:
I think a visual would go a long way here. https://github.com/mapbox/mapbox-gl-js-docs/pull/72 for your consideration. 💁

Reply:
Also -- I didn't update the text for `getRenderWorldCopies` and `setRenderWorldCopies` yet, but once we're comfortable with the language here I can update those, too.